packages/typespec-ts/src/index.ts

```diff
@@ -173,6 +173,17 @@ export async function $onEmit(context: EmitContext) {
   }
 
   // 5. Generate metadata and test files
+  let apiVersion: string | undefined;
+  for (const version of dpgContext.__clientToApiVersionClientDefaultValue.values()) {
```

Comment:
report diglost if api-version len > 1. return first.

multiple-api-version is not supported well.

Reply:
apiVesion === "all",